torch/__init__.py

```diff
@@ -1725,6 +1725,83 @@ def fn(model: Callable):
 
 from torch import export as export
 
+def cond(pred, true_branch, false_branch, operands):
```

Comment:
We shouldn't put this directly into torch/__init__.py, there should be a `from torch._ops._higher_order.control_flow import cond` or something in `torch/__init__.py`.

To pacify our public API tests, we need to set cond's `.__module__` to "torch"

Reply:
Ok! How to do the set cond's .__module__ to "torch"? I remember a exposed_in helper function in functorch. Can I use that?

Reply:
Yes, exposed_in